src/main/java/org/broadinstitute/hellbender/cmdline/SpecialArgumentsCollection.java

```diff
@@ -17,13 +17,13 @@
     public static final String ARGUMENTS_FILE_FULLNAME = "arguments_file";
```

Comment:
It's weird that this is in barclay and gatk.  I think this should either be deleted in gatk, or it should be deleted in barclay and this should be refactored to use special argument names that are exposed in barclay.